modules/introduction/partials/new-features-80.adoc

```diff
@@ -109,6 +109,50 @@ You can modify these services using the Couchbase xref:manage:manage-nodes/modif
+Before Couchbase Server 8.0, Magma only supported using 1024 vBuckets and had a minimum memory quota of 1{nbsp}GiB per node.
+Buckets that use Magma with 128 vBuckets have a minimum memory quota of 100{nbsp}MiB per node, which is the same as Couchstore.
+
++
+NOTE: These changes do not affect Couchbase Server Community Edition because it does not support the Magma storage engine.
+
++
+This change in the default storage engine does not affect existing buckets.
+You can still create buckets that use the Couchstore storage engine or the Magma storage engine with 1024 vBuckets by explicitly specifying them when you create the bucket.
+
++
+See xref:learn:buckets-memory-and-storage/storage-engines.adoc[] for more information about storage engines.
+
++
+.default behavior change
+[IMPORTANT] 
+====
+The new default storage backend for buckets is a behavior change that may cause issues if you rely on the previous behavior.
+
+Before upgrading to Couchbase Server 8.0, consider the following:
+
+* If you have deployment scripts that create buckets without specifying the storage engine, those scripts create Magma buckets with 128 vBuckets instead of Couchstore buckets after the upgrade.
+This may affect your deployment if you depend on buckets using the Couchstore storage engine.
+* If you have existing deployment scripts that create Magma buckets, in Couchbase Server 8.0 they create buckets with 128 vBuckets instead of the 1024 vBuckets they did previously. 
+To have them create the same Magma buckets with 1024 vBuckets they did prior to 8.0, set the number of vBuckets when creating the bucket using the `numVbuckets` parameter in xref:rest-api:rest-bucket-create.adoc#numvbuckets[the REST API] or `--num-vbuckets` in the xref:cli:cbcli/couchbase-cli-bucket-create.adoc[couchbase-cli command line tool]. 
+If you can allocate at least 1 GiB of memory for the Magma bucket, you should create a Magma bucket with 1024 vBuckets for better performance at scale.
+* You cannot use XDCR to replicate from Magma buckets on pre-8.0 clusters to Magma buckets using 128 vBuckets. 
+XDCR in pre-8.0 clusters only supports replication between buckets that contain the same number of vBuckets.
+Pre-8.0 versions of Couchbase Server only support Couchstore and Magma buckets with 1024 vBuckets (or 64 vBuckets on macOS).
+
++
+If you're manually deploying a new cluster, set the number of vBuckets to 1024 for any Magma bucket that is the target of a replication from a pre-8.0 cluster.
+See xref:manage:manage-buckets/create-bucket.adoc[] for more information about creating buckets.
+If you have a deployment script that creates Magma buckets that are an XDCR target for a pre-8.0 cluster, have it set the number of vBuckets to 1024.
+To set this value, use the `numVbuckets` parameter in xref:rest-api:rest-bucket-create.adoc#numvbuckets[the REST API] or `--num-vbuckets` in the xref:cli:cbcli/couchbase-cli-bucket-create.adoc[`couchbase-cli` command line tool].
```

Comment:
The parameter name is numVBuckets -- it is very important that customers use the exact spelling, so we should make sure that whenever the REST API parameter name is mentioned, that it has the correct spelling.

If it does not have the correct spelling when used, because it's an optional parameter, it will be silently ignored (a 202 will be returned), so the customer may not be aware that anything is wrong